### PR TITLE
fix(lvm): clean up whitespace in messages

### PR DESCRIPTION
**Pull request by teigland**
The lvm module has been printing messages with whitespace damage:

  Scanning devices sda sdb  for LVM logical volumes vg/root
  vg/swap

Remove the trailing space after the final device name, and remove the newlines between LV names.  The message above becomes:

  Scanning devices sda sdb for LVM logical volumes vg/root vg/swap

This pull request changes...

## Checklist
- [x ] I have tested it locally

### CODE REVIEW

**Comment by teigland** [comment]
github itself has destroyed whitespace in the original commit message that was illustrating the whitespace damage being fixed.  Being fixed is an extra space after the last device name, and a newline character after each LV name.

**Comment by teigland** [comment]
I replaced the branch with an updated version.  Looks like it might still need a shellcheck comment...

**Comment by aafeijoo-suse** [comment]
> I replaced the branch with an updated version. Looks like it might still need a shellcheck comment...

Yes, `# shellcheck disable=SC2027 disable=SC2086` before both lines. Or keeping your old code using bashisms, but replacing `#!/bin/sh` with `#!/bin/bash` on the first line. The final choice depends on the project admins' preferences.

File: modules.d/90lvm/lvm_scan.sh [hunk 1]
```diff
@@ -134,7 +134,7 @@
     && scan_args="$scan_args --nohints"
 
 if [ -n "$LVS" ]; then
-    info "Scanning devices $lvmdevs for LVM logical volumes $LVS"
+    info "Scanning devices "${lvmdevs% }" for LVM logical volumes $(echo $LVS)"
```
[the remark this answers is not in the document]
Comment (aafeijoo-suse) [reply modules.d/90lvm/lvm_scan.sh:137]: No need to surround `${lvmdevs% }` with quotes.

**Review by LaszloGombos — APPROVED** [review]
changes look good to me. We usually do not have merge commits.

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**